Change to DID Document and did:earth where appropriate throughout. Refactor heading nesting. (#14)
EarthProgram/did-earth: README.md

did:earth:1:ixo:nft:7Tqg6BwSSWapxgUDm9KKgg#myresource
```

### DID Documents

> A DID Document associated with an earth DID is a set of data describing a DID subject. The [representation of a DID Document when requested for production](https://www.w3.org/TR/did-core/#representations) MUST meet the DID Core specifications.

Reviewer: ```suggestion
A DID document associated with an earth DID is a set of data describing a DID subject. The [representation of a DID Document when requested for production](https://www.w3.org/TR/did-core/#representations) MUST meet the DID Core specifications.
```

}
```

> #### Elements for a W3C specification compliant DID Document representation

Reviewer: ```suggestion
#### Elements for a W3C DID Core Compliant DID Document Representation
```

In addition to using the JSON representation production rules, JSON-LD production MUST include the representation-specific @context entry. The serialized value of @context MUST be the JSON String https://www.w3.org/ns/did/v1, or a JSON Array where the first item is the JSON String https://www.w3.org/ns/did/v1 and the subsequent items are serialized according to the JSON representation production rules.
```

> #### Additonal elements specific to the did:earth method DID Document representation

Reviewer: ```suggestion
#### Additonal elements specific to the did:earth method DID document representation
```

21. **`displayDescription`** (optional): a property of Linked Resources that provide a longer text phrase for displaying additional detail about the asset.
22. **`displayIcon`** (optional): a property of Linked Resources that provide a URL for an image asset to use when displaying the asset.

> #### State format for DID Documents on ledger

Reviewer: ```suggestion
#### State format for DID documents on ledger
```

}
```

> #### DID Document metadata

Reviewer: ```suggestion
#### DID document metadata
```

> ##### Example of DID Document metadata

Reviewer: ```suggestion
##### Example of DID document metadata
```

**Note**: Verification method cannot contain both `publicKeyJwk` and `publicKeyMultibase` but must contain at least one of them.

> ##### Example of Verification method in a DID Document

Reviewer: ```suggestion
##### Example of Verification method in a DID document
```

}
```
#### Service

> Services can be defined in a DID Document to express means of communicating with the DID subject or associated entities.

Reviewer: ```suggestion
Services can be defined in a DID document to express means of communicating with the DID subject or associated entities.
```

> 1. **`id`** (string): The value of the `id` property for a Service MUST be a URI conforming to [RFC3986](https://www.rfc-editor.org/rfc/rfc3986). A conforming producer MUST NOT produce multiple service entries with the same ID. A conforming consumer MUST produce an error if it detects multiple service entries with the same ID. It has a follow formats: `<did-document-id>#<service-alias>` or `#<service-alias>`.

Reviewer: ```suggestion
1. **`id`** (string): The value of the `id` property for a Service MUST be a URI conforming to [RFC3986](https://www.rfc-editor.org/rfc/rfc3986). A conforming producer MUST NOT produce multiple service entries with the same ID. A conforming consumer MUST produce an error if it detects multiple service entries with the same ID. The ID defines a node in the JSON-LD graph of the DID document, using either `<did-of-this-document>#<service-alias>` or `#<service-alias>`.
```

2. **`type`** (string): The service type and its associated properties SHOULD be registered in the [DID Specification Registries](https://www.w3.org/TR/did-spec-registries/)
3. **`serviceEndpoint`** (strings): A string that conforms to the rules of [RFC3986](https://www.rfc-editor.org/rfc/rfc3986) for URIs, a map, or a set composed of a one or more strings that conform to the rules of
[RFC3986](https://www.rfc-editor.org/rfc/rfc3986) for URIs and/or maps.

> ##### Example of Service in a DID Document

Reviewer: ```suggestion
##### Example of Service in a DID document
```

The operation CAN be executed by anyone and is publicly available.

- **`id`**: `QueryIdentifierDocument` should be a fully qualified DID of type `did:earth:<chainspace>:<namespace>`. It MUST be the DID that is to be resolved. Allowed `chainspace` and `namespace` values are available in the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry)

> - **`metadata`**: Contains DID Document metadata? `created`, `updated`, `valid`, `versionId`

Reviewer: ```suggestion
- **`metadata`**: Contains DID document metadata? `created`, `updated`, `valid`, `versionId`
```
